packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -390,6 +393,272 @@ class ToolbarGroupTypes extends React.Component {
+              <SelectOption isDisabled={option.disabled} key={index} value={option.value}>
+                <Badge key={index}>C</Badge>
+                {` ${option.value}`}
+              </SelectOption>   
+            ))}
+          </Select>
+        </ToolbarItem>
+        <ToolbarItem>
+          <Button variant="link" onClick={this.pauseOrStart}>
+          {this.state.isPaused? <PlayIcon /> : <PauseIcon />}
+          {` Log Stream`}
+          </Button>
+        </ToolbarItem>
+        <ToolbarItem>
+          <Switch
+            id="switch-1"
+            label="Timestamp"
+            isChecked={firstSwitchChecked}
+            onChange={this.handleFirstSwitchChange}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
+          <Switch
+            id="switch-2"
+            label="Wrap lines"
+            isChecked={secondSwitchChecked}
+            onChange={this.handleSecondSwitchChange}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
+          <SearchInput 
+            placeholder='find'
+            value={this.state.searchValue}
+            onChange={this.onSearchChange}
+            onClear={this.onSearchClear}
+            resultsCount={`${this.state.currentSearchResult} / ${this.state.searchResultsCount}`}
+            onNextClick={this.onSearchNext}
+            onPreviousClick={this.onSearchPrevious}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
+          <Dropdown
+            onSelect={this.onSecondSelect}
+            toggle={
+              <DropdownToggle
+                splitButtonItems={[
+                  <DropdownToggleAction key="action" onClick={this.onSecondClick}>
+                    External Logs
+                  </DropdownToggleAction>
+                ]}
+                splitButtonVariant="action"
+                onToggle={this.onSecondToggle}
+              />
+            }
+            isOpen={secondIsExpanded}
+            dropdownItems={firstDropdownItems}
+          />
+        </ToolbarItem>
+         <ToolbarItem>
+          <Dropdown
+            onSelect={this.onThirdSelect}
+            toggle={
+              <DropdownToggle
+                splitButtonItems={[
+                  <DropdownToggleAction key="action" onClick={this.onThirdClick}>
+                    Download
+                  </DropdownToggleAction>
+                ]}
+                splitButtonVariant="action"
+                onToggle={this.onThirdToggle}
+              />
+            }
+            isOpen={thirdIsExpanded}
+            dropdownItems={secondDropdownItems}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
+          <Button variant="link" aria-label="expand">
+            <ExpandIcon />
+          </Button>
+        </ToolbarItem>
+      </React.Fragment>
+    );
+
+    return (
+      <Toolbar id="toolbar-group-types">
+        <ToolbarContent>{items}</ToolbarContent>
+      </Toolbar>
+    );
+  }
+}
+```
+
```

Comment:
We also need to take into account the compressed version of this component. The design located here: http://openshift.github.io/openshift-origin-design/designs/administrator/future-openshift/log%20toolbars/#compressed-toolbar

Reply:
I would like to discuss how the users would like to use the compressed version in tomorrow's meeting, so I would implement this after tomorrow.

Reply:
I do believe that you will make use of the `ToolbarToggleGroup` and/or the `OverflowMenu`
I'm happy to help with this, too if need be. I'm not as familiar with the `OverflowMenu` but there may be opportunity for multiple of them in this case.